src/main/java/com/minecolonies/blockout/controls/ButtonHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.minecolonies.blockout.controls;
+
+/**
+ * Used for windows that have buttons and want to respond to clicks.
+ */
+@FunctionalInterface
+public interface ButtonHandler
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Drop this interface in favor of "java.util.function.Consumer<Button>". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1711)